concrete/bootstrap/configure.php

```diff
@@ -305,7 +305,6 @@
 const GUEST_GROUP_ID = '1';
 const REGISTERED_GROUP_ID = '2';
 const ADMIN_GROUP_ID = '3';
-const USER_FOREVER_COOKIE_LIFETIME = 1209600; // 14 days
```

Comment:
I'd keep this constant (some package may rely on it), but marking it as `@deprecated` (with a comment stating that the configuration key should be used)

Reply:
Updated.